content/pt-br/docs/reference/glossary/kops.md

```diff
@@ -0,0 +1,28 @@
+---
+title: KOps (Kubernetes Operations)
+id: kops
+date: 2018-04-12
+full_link: /pt-br/docs/getting-started-guides/kops/
+short_description: >
+  kOps não só te ajudará a criar, destruir, atualizar e manter clusters Kubernetes em nível de produção e altamente disponíveis, mas também fornecerá a infraestrutura de nuvem necessária.
+
+aka: 
+tags:
+- tool
+- operation
+---
+  `kOps` não só te ajudará a criar, destruir, atualizar e manter clusters Kubernetes em nível de produção e altamente disponíveis, mas também fornecerá a infraestrutura de nuvem necessária.
```

Comment:
nit: I think we could use "provisionará" instead of "fornecerá" (compare provision to provides).